src/frida/repl.py

```diff
@@ -67,6 +81,8 @@ def set_return():
                 pass
 
         def _stop(self):
+            if self._script_monitor != None:
```

Comment:
For consistency this should be written as:
`if self._script_monitor is not None:`